python/py-kubernetes/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kubernetes
+version             17.17.0
+revision            0
+
+homepage            https://github.com/kubernetes-client/python
+
+description         Official Python Client Library for Kubernetes
+
+long_description    ${description}
+
+license             Apache-2.0
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+supported_archs     noarch
+categories-append   sysutils
+
+python.versions     39
+
+checksums           rmd160  08aa86d29509c4cf0727b3db7f8542b5fb8b1d27 \
+                    sha256  c69b318696ba797dcf63eb928a8d4370c52319f4140023c502d7dfdf2080eb79 \
+                    size    869970
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-certifi \
+                    port:py${python.version}-dateutil \
+                    port:py${python.version}-google-auth \
+                    port:py${python.version}-ipaddress \
```

Comment:
```suggestion
```

Thanks @herbygillot for adding this port.

Just looking at the [requirements.txt](https://github.com/kubernetes-client/python/blob/master/requirements.txt#L7), I don't think we require py-ipaddress for py39:

```txt
ipaddress>=1.0.17;python_version=="2.7"  # PSF
```

(Added in https://github.com/kubernetes-client/python/commit/e2b7ee7c9ee594008894649d2cbcd5080bdd7f62)

Also, it might be beneficial to add a `platforms` attribute. Otherwise, LGTM 👍 Feel free to merge when ready.

Reply:
Thank you @harens, changes made as suggested